### PR TITLE
Fixes #287 - Adds filters to LoklakWordCloud app

### LINKED ISSUE #287: Add filters to Loklak wordcloud app
Add filters to LoklakWordCloud app. Using these filters users should be able to plot a word cloud only based on hashtags or mentions or tweet text or all of them.

### PR DESCRIPTION
**Pull request by djmgit**
Fixes issue #287, added filters for hashtags, mentions, tweet text
and dates.
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link: http://loklakapps.surge.sh/LoklakWordCloud/

@singhpratyush @kavithaenair @SKrPl please review, I have implementef  here - http://loklakapps.surge.sh/LoklakWordCloud/
please revuew

File: LoklakWordCloud/js/script.js [hunk 1]
```diff
@@ -21,64 +41,68 @@ app.controller("app", function ($scope, $http) {
...
+                        continue;
+                    }
+                    word = word.trim();
+                    if (stopwords.indexOf(word.toLowerCase()) !== -1) {
+                        continue;
+                    }
+                    if (word.startsWith("#") || word.startsWith("@")) {
+                        continue;
+                    }
+                    if (word.startsWith("http") || word.startsWith("https")) {
```
Comment (singhpratyush) [inline LoklakWordCloud/js/script.js]: Only the first check should do.

File: LoklakWordCloud/js/script.js [hunk 2]
```diff
@@ -21,64 +41,68 @@ app.controller("app", function ($scope, $http) {
...
-                   hashtags and mentions are handles separately
-                 */
-                if (word.indexOf("#") !== -1) {
-                    word = word.split("#")[0];
-                }
-                if (word.indexOf("@") !== -1) {
-                    word = word.split("@")[0];
+                    /* following two conditions handles cases where there is no delimiter between
+                       a normal word and a hashtag or mention. We choose only the first word since
+                       hashtags and mentions are handles separately
```
Comment (singhpratyush) [inline LoklakWordCloud/js/script.js]: handled*

**Comment by djmgit** [comment]
@singhpratyush did the changes, please re-review.

**Pull request merged**